### PR TITLE
Fix #12810 - Missing docs for ingress setup on minikube

### LINKED ISSUE #12810: Missing docs for ingress setup on minikube
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**

There is a `404 NotFound` page after clicking the last link in the [ingress documentation file](https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/services-networking/ingress.md). I mean this one - `Set up Ingress on Minikube with the NGINX Controller`. 

There is no such file as `https://github.com/kubernetes/website/blob/master/docs/tasks/access-application-cluster/ingress-minikube.md`.

**Proposed Solution:**

Create the file with ingress setup on minikube.

**Page to Update:**

https://kubernetes.io/docs/concepts/services-networking/ingress/

### PR DESCRIPTION
**Pull request by daminisatya**
#12810 

Fix One - Removed .md from the URL
Fix Two - the YAML formatting is correctly shown. here are the screenshots before and after the changes. 

Before - 

screen shot 1940-12-06 at 4 35 19 pm

After fix -

screen shot 1940-12-06 at 4 25 14 pm

### CODE REVIEW

File: content/en/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 1]
```diff
@@ -160,19 +159,19 @@ The following file is an Ingress resource that sends traffic to your Service via
     ```
 
 1. Create the Ingress resrouce by running the following command:
-
```
Comment (kbhawkey) [inline content/en/docs/tasks/access-application-cluster/ingress-minikube.md]: Good updates.
nit: correct the spelling of resource in line 161.

File: content/en/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 2]
```diff
@@ -195,9 +194,9 @@ The following file is an Ingress resource that sends traffic to your Service via
 
 1. Verify that the Ingress contorller is directing traffic:
```
Comment (kbhawkey) [inline content/en/docs/tasks/access-application-cluster/ingress-minikube.md]: nit: line 195, correct spelling of controller
nit: should this be item number 5? Probably an extra line needs to be taken away or added.

File: content/en/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 3]
```diff
@@ -252,7 +251,7 @@ The following file is an Ingress resource that sends traffic to your Service via
     kubectl apply -f example-ingress.yaml
     ```
```
Comment (kbhawkey) [inline content/en/docs/tasks/access-application-cluster/ingress-minikube.md]: nit: check item ordering (1, 2 vs 1, 1). May need to add/delete extra line.

**Comment by daminisatya** [comment]
@kbhawkey Thank you so much for the review! I will update the PR right away.

**Comment by daminisatya** [comment]
@kbhawkey Addressed all the issues!

**Comment by zparnold** [comment]
@daminisatya Could I please have you fix the merge conflicts, otherwise I think this is ready to go!

File: content/en/docs/concepts/services-networking/ingress.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ uses a service of type [Service.Type=NodePort](/docs/concepts/services-networkin
 
 {{< feature-state for_k8s_version="v1.1" state="beta" >}}
 
-Before you start using an Ingress, there are a few things you should understand. The Ingress is a beta resource. 
+Before you start using an Ingress, there are a few things you should understand. The Ingress is a beta resource.
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/ingress.md]: I'm assuming that Ingress has left beta; we're long past v1.1. Is that right?

Comment (daminisatya) [reply content/en/docs/concepts/services-networking/ingress.md]: This needs to be confirmed before it's updated in the document. Also, I don't think it is related to the current. We can open another bug to track it.

**Comment by daminisatya** [comment]
@zparnold This has been fixed by this PR. https://github.com/kubernetes/website/pull/12720

**Pull request closed without merging**